keyboards/bananasplit/readme.md

```diff
@@ -0,0 +1,28 @@
+Bananasplit keyboard firmware
```

Comment:
Looks like this file has been duplicated, maybe due to casing? A "README.md" already exists.